src/main/java/org/springframework/integration/kafka/inbound/KafkaMessageDrivenChannelAdapter.java

```diff
@@ -297,7 +299,7 @@ public int afterShutdown() {
 		}
 
 		@Override
-		public void onMessage(ConsumerRecord<K, V> record, Acknowledgment acknowledgment) {
+		public void onMessage(ConsumerRecord<K, V> record, Acknowledgment acknowledgment, Consumer<?, ?> consumer) {
```

Comment:
Are we going to populate `consumer` in to the `headers` like we do with the AMQP `Channel` ?

Otherwise LGTM at a glance